package/MDAnalysis/topology/PDBParser.py

```diff
@@ -95,6 +97,29 @@
 # Set up a logger for the PDBParser
 logger = logging.getLogger("MDAnalysis.topology.PDBParser")
 
+try:
+    import pooch
+except ImportError:
+    HAS_POOCH = False
+else:
+    HAS_POOCH = True
+
+DEFAULT_CACHE_NAME_DOWNLOADER = "MDAnalysis_pdbs"
```

Comment:
```suggestion
#: Name of the :mod:`pooch` cache directory ``pooch.os_cache( DEFAULT_CACHE_NAME_DOWNLOADER)``;
#: see :func:`pooch.os_cache` for further details.
DEFAULT_CACHE_NAME_DOWNLOADER = "MDAnalysis_pdbs"
```

You then also need to add a
```reST
.. autodata:: DEFAULT_CACHE_NAME_DOWNLOADER
```
into the docs and update `conf.py` with the correct [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) links https://github.com/MDAnalysis/mdanalysis/blob/5c7c4805f6c114dc0dfb75674e66c18d102a5790/package/doc/sphinx/source/conf.py#L334-L353
so that the func/mod roles resolve: Add
```python
    "pooch": ("https://www.fatiando.org/pooch/latest/", None),
```
as the last entry after imdclient.

You can the refer to `DEFAULT_CACHE_NAME_DOWNLOADER` in reST as
```reST
The default cache name is taken from :data:`DEFAULT_CACHE_NAME_DOWNLOADER`.
```